tests/functional/n/names_in__all__.py

```diff
@@ -27,7 +27,7 @@ class Dummy:
     """A class defined in this module."""
     pass
 
-DUMMY = Dummy()
+dummy = Dummy()
```

Comment:
I don't really understand what you're changing in this PR, but this change is incorrect.

`DUMMY` is a "constant" in `pylint`/`Python` terms and should be uppercased.

Reply:
> `DUMMY` is a "constant" in `pylint`/`Python` terms and should be uppercased.

I agree with Daniel here. PEP 8 never differentiates between a constants and (true) constants. The pylint interpretation has been "consider every variable at module level a constant" for a long time. Changing it now would result in a massive breaking change that's just unreasonable.

Just an example. The change would result in over 7500+ `invalid-name` errors for Home-Assistant.

I'm strong -1.